examples/evaluate_a_judge_model_capabilities_on_arena_hard.py

```diff
@@ -0,0 +1,35 @@
+from unitxt import evaluate, load_dataset
+from unitxt.inference import MockInferenceEngine
+from unitxt.text_utils import print_dict
+
+model_id = "meta-llama/llama-3-70b-instruct"
+model_format = "formats.llama3_instruct"
+
+"""
+We are evaluating only on a small subset (by using "select(range(4)), in order for the example to finish quickly.
+The dataset full size if around 40k examples. You should use around 1k-4k in your evaluations.
+"""
+dataset = load_dataset(
+    f"card=cards.arena_hard.response_assessment.pairwise_comparative_rating.both_games_gpt_4_judge,"
+    "template=templates.response_assessment.pairwise_comparative_rating.arena_hard_with_shuffling,"
+    f"format={model_format}"
+)["test"].select(range(4))
```

Comment:
I think it's better not to use strings , but instead :

taset = load_dataset(
    card="cards.arena_hard.response_assessment.pairwise_comparative_rating.both_games_gpt_4_judge",
    template="templates.response_assessment.pairwise_comparative_rating.arena_hard_with_shuffling"
    format=model_format,
    max_test_instances=4
)["test"]

Reply:
Oh ok. Didn't know it's possible (the interface doesn't tell you that :X).
btw notice that we cannot use max_test_instances=4
We have to load all the dataset because of the join operation